docs/modules/reference/pages/provisioning/policies.adoc

```diff
@@ -10,6 +10,19 @@ For information on how to extend the functionality of these policies, see the xr
 == Common policy parameters
 
 The matchBehavior action is a boolean control that determines how the optional parameters will be evaluated.
-Setting this parameter’s value to `ALL_PARAMETERS` causes Provisiond to evaluate each optional parameter with boolean AND logic and the value `ANY_PARAMETERS` causes OR logic to be applied.
+
+ALL_PARAMETERS::
+    All defined parameters must match (AND Boolean logic) for the policy to apply.
+ANY_PARAMETERS::
+    Any defined parameters can match (OR Boolean logic) for the policy to apply.
+NO_PARAMETERS::
+    No defined parameters can match (opposite of ANY_PARAMETERS) for the policy to apply.
```

Comment:
Probably a nitpick, but I think "Boolean \<operator\> logic" would be a bit clearer.

```suggestion
ALL_PARAMETERS::
    The policy applies when all defined parameters match (Boolean AND logic).
ANY_PARAMETERS::
    The policy applies when at least one set of defined parameters matches (Boolean OR logic).
NO_PARAMETERS::
    The policy applies when no defined parameters match (opposite of ANY_PARAMETERS).
```